bot/exts/utilities/bookmark.py

```diff
@@ -105,6 +109,26 @@ def build_error_embed(message: str) -> discord.Embed:
             colour=Colours.soft_red,
         )
 
+    async def _delete_bookmark_context_menu_callback(
+            self,
+            interaction: discord.Interaction,
+            message: discord.Message
+    ) -> None:
```

Comment:
his shouldn't be double-indented (maybe add a trailing comma while you're editing this too)

Reply:
Done in 6047ad7165a16ae499d95766ad3dfc20a64c5a63